openedx/core/djangoapps/models/config/waffle.py

```diff
@@ -0,0 +1,39 @@
+"""
+This module contains various configuration settings via
+waffle switches for the course_details view.
+"""
+
+
+from openedx.core.djangoapps.waffle_utils import (
+    CourseWaffleFlag,
+    WaffleFlagNamespace,
+    WaffleSwitchNamespace
+)
+
+COURSE_DETAIL_WAFFLE_NAMESPACE = u'course_detail'
+COURSE_DETAIL_WAFFLE_FLAG_NAMESPACE = WaffleFlagNamespace(name=COURSE_DETAIL_WAFFLE_NAMESPACE)
+WAFFLE_SWITCHES = WaffleSwitchNamespace(name=COURSE_DETAIL_WAFFLE_NAMESPACE)
+
+# Course Override Flag
+COURSE_DETAIL_UPDATE_CERTIFICATE_DATE = u'course_detail_update_certificate_date'
+
+
+def waffle_flags():
+    """
+    Returns the namespaced, cached, audited Waffle flags dictionary for course detail.
+    """
+    return {
+        COURSE_DETAIL_UPDATE_CERTIFICATE_DATE: CourseWaffleFlag(
+            waffle_namespace=COURSE_DETAIL_WAFFLE_NAMESPACE,
+            flag_name=COURSE_DETAIL_UPDATE_CERTIFICATE_DATE,
+            flag_undefined_default=False,
+        )
+    }
+
+
+def course_detail_update_certificate_date(course_id):
```

Comment:
Change the name to indicate if the feature is enabled or not. This name doesn't portray the underlying functionality.